model/src/main/kotlin/licenses/LicenseInfoResolver.kt

```diff
@@ -117,6 +117,25 @@ class LicenseInfoResolver(
                         )
                     )
                 }
+
+                licenseInfo.declaredLicenseInfo.copyrightHolders.takeIf { it.isNotEmpty() }?.let {
```

Comment:
Nit: `let` is usually used when the return value matters; maybe use `also` instead?

Reply:
Done